### ISSUE TITLE
displaying pages full of just one category

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#54]
```
How do i refer to a custom field in a custom content type in a url? as in, make 
a url to display all the "Action" entries from the "Films" custom type?

I'm generally having issues with the permalinks - as in they don't seem to work 
hehe. Looking forward to 0.88!

Chris

What version of the plugin are you using? 0.86&0.85
What version of WordPress? 3.1

Please provide any additional information below (e.g. Browser, version,
OS):

```

Original issue reported on code.google.com by `bigonr...@gmail.com` on 3 Apr 2011 at 7:32

**Comment by GoogleCodeExporter** [comment]
```
Sorry, just to give more info on the permalinks: 
* with /postname setting on, I get a 404 error on visiting the page
* same with custom. 

Query changing works though - so ?shop=name
```

Original comment by `bigonr...@gmail.com` on 3 Apr 2011 at 7:47

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the bug report!  

I've had a crazy time trying to figure out how WP handles permalinks.  I've 
found that the safest way by far to work with customized post types is to leave 
them off, but if your .htaccess file is correctly edited and permalinks are 
working for your normal posts, then they should work for the custom post types 
as well.  I've found a couple other plugins that conflict with this 
functionality -- do you have any other plugins running?

I am working hard to get the next version done -- it's a huge improvement 
architecturally, but I haven't spear-headed the permalink stuff yet.
```

Original comment by `fireproofsocks` on 4 Apr 2011 at 2:15
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Reading this more thoroughly, if you assume that the 404 issues are a separate 
issue...

It depends on how you want this to work, but I would recommend checking out my 
other plugin: Summarize Posts.  
http://code.google.com/p/wordpress-summarize-posts/

It has all sorts of options available for you to search for posts.  I think 
this is more of a template question, as in how to index (i.e. summarize) posts 
of one type.  In your example, is "Action" the value of a dropdown custom 
field?  If so, you might want to look at the custom template functions here: 
http://code.google.com/p/wordpress-custom-content-type-manager/wiki/TemplateFunc
tions

In particular, have a look at the get_posts_sharing_custom_field_value() 
function.  It may be exactly what you need.  

I'm marking this closed for now so I can get 0.8.8 out tonight.  Please post 
more data if this does not resolve your issue.

```

Original comment by `fireproofsocks` on 8 Apr 2011 at 12:41
- Changed state: **Fixed**
